### PR TITLE
feat(locale): add `state` and `state_abbr` to `cz`

### PR DESCRIPTION
**Pull request by xDivisionByZerox**
Fill the `states` dataset for the locale `cz`.
In the  Czech Republic "states" are called "regions". They are not the same but serve the same purpose for our data.

---

- [source I used](https://en.wikipedia.org/wiki/ISO_3166-2:CZ#Regions)
- [More information on Czech administrative divisions](https://en.wikipedia.org/wiki/Czech_Republic#Administrative_divisions)

### CODE REVIEW

**Comment by xDivisionByZerox** [comment]
I was not able to find anything regarding state abbreviations. Should I simply copy the existing dataset in that case?

File: test/all_functional.spec.ts [hunk 1]
```diff
@@ -20,13 +20,13 @@ function isMethodOf(mod: string) {
 }
 
 const BROKEN_LOCALE_METHODS = {
-  // TODO ST-DDT 2022-03-28: these are TODOs (usually broken locale files)
+  // TODO ST-DDT 2022-03-28: these are TODOs (usually broken locale tschechienfiles)
```
Comment (matthewmayer) [inline test/all_functional.spec.ts]: was this line changed by mistake?

**Review by matthewmayer — DISMISSED** [review]

**Comment by ST-DDT** [comment]
> I was not able to find anything regarding state abbreviations. Should I simply copy the existing dataset in that case?

Instead of copying it you can import it and export it afterwards (with a comment why).

File: test/all_functional.spec.ts (hunk 1 above)
Comment (xDivisionByZerox) [reply test/all_functional.spec.ts]: Yes, probably forgot to focus my browser while copy passing or something 🤔 
I'll remove this.

**Review by ST-DDT — APPROVED** [review]

**Review by matthewmayer — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**